ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -35,6 +37,7 @@ basic_tests() {
                        --kubeconfig "${KUBECONFIG}" \
                        --verbose \
                        --output json \
+                       --wait-time 300s \
```

Comment:
300s as wait time seems excessive. I was hoping we'd just double the time to 60s. The current default is 30s if you don't specify any waitTime and that seems to work for the most part.

Reply:
resolved